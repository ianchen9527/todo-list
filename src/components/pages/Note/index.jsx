import React from "react";
import styles from "./index.module.scss";
import NoteList from "../../../containers/NoteList";
import NoteIncreaser from "../../../containers/NoteIncreaser";
import NoteViewer from "../../../containers/NoteViewer";

const Note = () => {
  return (
    <div className={styles["wrapper"]}>
      <div className={styles["note-increaser-wrapper"]}>
        <NoteIncreaser />
      </div>
      <div className={styles["note-list-wrapper"]}>
        <NoteList />
      </div>
      <div className={styles["note-viewer-wrapper"]}>
        <NoteViewer />
      </div>
    </div>
  );
};

export default Note;
