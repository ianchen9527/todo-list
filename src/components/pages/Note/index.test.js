import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Note from './index';

describe('Notes', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Note />);
  });

  it('should render without crashing', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
