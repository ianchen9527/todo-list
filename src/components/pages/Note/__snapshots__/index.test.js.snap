// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes should render without crashing 1`] = `
<div
  className="wrapper"
>
  <div
    className="note-increaser-wrapper"
  >
    <withRouter(Connect(NoteIncreaser)) />
  </div>
  <div
    className="note-list-wrapper"
  >
    <withRouter(Connect(withLifecycle(NoteList))) />
  </div>
  <div
    className="note-viewer-wrapper"
  >
    <withRouter(Connect(withLifecycle(NoteViewer))) />
  </div>
</div>
`;
